CONTRIBUTING.md

```diff
@@ -146,6 +146,14 @@ Now it's time to fill in these files!
   removal of public fields were to report that a struct was removed.
   Struct removal has its own lint, so there's no reason to also report that
   the removed struct also had its fields removed.
+- Note that your lint must output `span_filename` and `span_begin_line` for it
+  to be a valid lint. The pattern we commonly use is:
+  ```
+  span_: span @optional {
+    filename @output
+    begin_line @output
+  }
+  ```
```

Comment:
Do you think this note should be here, or next to the "define the lint" part? Is this where it seems more intuitive? I have the contributing steps in muscle memory so I genuinely don't know, happy to leave it here if this is where you think it's best.

Reply:
Yup, this should be moved